doc/specs/#2046 - Command Palette.md

```diff
@@ -248,6 +287,96 @@ For example, consider the following list of commands:
 As the user types, we should **bold** each matching character in the command
 name, to show how their input correlates to the results on screen.
 
+Additionally, it will be important for commands in the action list to display
+the keybinding that's bound to them, if there is one.
+
+### Commandline Mode
+
+Commandline mode is much simpler. In this mode, we'll simply display a text input,
+similar to the search box that's rendered for Action Mode. In this box, the
+user will be able to type a `wt.exe` style commandline. The user does not need
+to start this commandline with `wt` (or `wtd`, etc) - since we're already
+running in WT, the user shouldn't really need to repeat themselves.
+
+When the user hits <kbd>enter</kbd>, we'll attempt to parse the commandline. If

```

Comment:
When we do add the multiprocess management, does the need for commandline mode go away since we could just dispatch the thing to CreateProcess and let the multiprocess behavior take over as appropriate?

Reply:
Presumably yes, but we'd probably leave it as a shorthand for `wt --in-this-window ...` (or whatever the arg end up being). Plus, I really like the idea that the Commandline Mode could give you real-time feedback on any parse errors (which I believe I left as a future consideration)